.github/skills/generate-vividus-api-tests/SKILL.md

```
@@ -128,35 +172,82 @@ When I set request headers:
+**Re-executability Rule**: Tests MUST NOT hardcode IDs or unique values in request bodies. Always use generated or dynamic values (e.g., `#{generate(...)}`, timestamps, random strings) to ensure tests can be re-executed without conflicts.
+
+❌ **Bad** - hardcoded values prevent re-execution:
 
 ```gherkin
-Given request body: {"name": "Test User", "email": "test@example.com"}
+Given request body: {"id": 999, "name": "Test User", "email": "test@example.com"}
+```
+
+✅ **Good** - dynamic values inside a composite step ensure re-executability and DRY:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)